bin/llvm-krun

```diff
@@ -14,7 +14,7 @@ cleanup () {
   rm -rf "$input_file" "$expanded_input_file" "$output_file" "$parser_file" "${temp_inputs[@]}"
 }
 trap 'cleanup' INT TERM EXIT
-
+set -x
```

Comment:
I don't think you want to leave this in globally!

Reply:
You're right! I forgot to clean it up! Sorry and thanks for reminding me!